content/docs/ops/managing-users.md

```diff
@@ -36,44 +36,44 @@ cf services
+2. Remove user's permissions to all [spaces and orgs](https://docs.cloudfoundry.org/adminguide/cli-user-management.html#orgs-spaces) other than their sandbox. Search for the user in the Admin interface to locate the relevant orgs and spaces.
+
+    For those spaces and orgs, notify the Space Managers and Org Managers that we've removed the user's access because of their request to reset their account's authentication application.
 
-    For those spaces and orgs, notify the Space Managers and Org Managers that we've removed the user's access because of their request to reset their account's authentication application. 
-    
 3. Reset the user's totp_seed in cloudfoundry's uaa database.
 
     Login to a **[concourse jumpbox]({{< relref "docs/ops/runbook/troubleshooting-bosh.md#creating-and-intercepting-ephemeral-jumpboxes" >}})**, connect to the appropriate DB, and remove the user, as in this example:
-    
-    ```sh
-    root@PRODUCTION:/tmp/build/8e72821d$ bosh -d cf-production manifest | grep -A7 uaadb
-      uaadb:
-	address: production.dns.name.aws.gov
-	databases:
-	- name: uaadb
-	  tag: uaa
-	db_scheme: postgresql
-	port: 5432
-	roles:
-	- name: cfdb
-	  password: secret_password
-	  tag: admin
-      
-    root@PRODUCTION:/tmp/build/8e72821d$ psql postgres://cfdb:secret_password@production.dns.name.aws.gov
-
-    uaadb=> select * from totp_seed where username='pat.jones@agency.gov';
-	  username              |       seed         | backup_code
-    ----------------------+--------------------+-------------
-     pat.jones@agency.gov | EAAS9HANFSD90ENADF |
-    (1 row)
-
-    uaadb=> begin;
-    BEGIN
-    uaadb=> delete from totp_seed where username='pat.jones@agency.gov';
-    DELETE 1
-    uaadb=> commit;
-    COMMIT
-    uaadb=>
-    ```
+
+ ```sh
+ root@PRODUCTION:/tmp/build/8e72821d$ bosh -d cf-production manifest | grep -A7 uaadb
+   uaadb:
+      address: production.dns.name.aws.gov
+      databases:
+      - name: uaadb
+      tag: uaa
+      db_scheme: postgresql
+      port: 5432
+      roles:
+      - name: cfdb
+      password: secret_password
+      tag: admin
+
+root@PRODUCTION:/tmp/build/8e72821d$ psql postgres://cfdb:secret_password@production.dns.name.aws.gov/uaadb
+
+uaadb=> select * from totp_seed where username='pat.jones@agency.gov';
+   username             |       seed         | backup_code
+   ---------------------+--------------------+-------------
+   pat.jones@agency.gov | EAAS9HANFSD90ENADF |
+   (1 row)
+
+uaadb=> begin;
+   BEGIN
+uaadb=> delete from totp_seed where username='pat.jones@agency.gov';
+   DELETE 1
+uaadb=> commit;
+   COMMIT
+uaadb=>
+```
 
 4. Let the user know the reset process is complete, so they can set up a new authentication application and request access from Space Managers and Org Managers again.
```

Comment:
could we change these from `1, 2, 3, 4` sequential numbering to `1, 1, 1, 1` instead. Markdown will take care of numbering them appropriately for us and it'll help in the maintenance in case we ever need to rearrange these.